ProjectLighthouse.Servers.Website/Pages/UsersPage.cshtml.cs

```diff
@@ -37,8 +37,9 @@ public async Task<IActionResult> OnGet([FromRoute] int pageNumber, [FromQuery] s
 
         if (this.PageNumber < 0 || this.PageNumber >= this.PageAmount) return this.Redirect($"/users/{Math.Clamp(this.PageNumber, 0, this.PageAmount - 1)}");
 
-        this.Users = await this.Database.Users.Where(u => u.PermissionLevel != PermissionLevel.Banned && u.Username.Contains(this.SearchValue))
-            .Where(u => u.ProfileVisibility == PrivacyType.All) // TODO: change check for when user is logged in
+        this.Users = await this.Database.Users
```

Comment:
Should people with private profiles be included in this list?

Reply:
This was done on purpose - since you should still be able to search for a user, as the information that would be shown there is the same thing shown on the user page anyways regardless of privacy settings.